buildbot_nix/__init__.py

```diff
@@ -33,13 +32,12 @@
 from .common import (
     slugify_project_name,
 )
-from .gitea_projects import GiteaBackend, GiteaConfig
+from .gitea_projects import GiteaBackend
 from .github_projects import (
     GithubBackend,
-    GithubConfig,
 )
+from .models import BuildbotNixConfig
```

Comment:
@MagicRB I think you forgot to commit the models module

Reply:
Did I? I did....

Reply:
it's in